Missing dep in installed headers: ICpuOperator.h -> "src/cpu/IcpuOperator.h"

I'm working on building https://github.com/uxlfoundation/oneDNN from source w/ ACL support. I used their [build script](https://github.com/uxlfoundation/oneDNN/blob/main/.github/automation/aarch64/build_acl.sh) as a reference (and tried Debian's parameters as well), but oneDNN fails to build with a missing header:

```
2025/09/03 14:40:26 INFO In file included from /home/build/src/cpu/aarch64/matmul/acl_lowp_matmul.hpp:22,
2025/09/03 14:40:26 INFO                  from /home/build/src/cpu/aarch64/matmul/acl_lowp_matmul.cpp:17:
2025/09/03 14:40:26 INFO /usr/include/arm_compute/runtime/experimental/operators/CpuQuantize.h:34:10: fatal error: src/cpu/ICpuOperator.h: No such file or directory
2025/09/03 14:40:26 INFO    34 | #include "src/cpu/ICpuOperator.h"
2025/09/03 14:40:26 INFO       |          ^~~~~~~~~~~~~~~~~~~~~~~~
2025/09/03 14:40:26 INFO compilation terminated.
```

I'm building v52.4.0 with the following command:
```
scons -j$(nproc) Werror=0 asserts=0 neon=1 opencl=0 embed_kernels=0 os=linux arch=armv8.2-a build=native multi_isa=1 fixed_\
format_kernels=1 cppthreads=0 openmp=1 examples=0 validation_tests=0 install_dir=install
```

Indeed, `CpuQuantize.h` does land in the install_dir, but `ICpuOperator.h` does not.

For now I'm working around it by manually copying those headers into the install root.

--- a/arm_compute/runtime/experimental/operators/CpuQuantize.h
+++ b/arm_compute/runtime/experimental/operators/CpuQuantize.h
@@ -31,8 +31,6 @@
 #include "arm_compute/core/ITensorInfo.h"
 #include "arm_compute/runtime/NEON/INEOperator.h"
 
-#include "src/cpu/ICpuOperator.h"
-
 #include <memory>
 
 /*

